### PR TITLE
Update from Intelsdi-x pluginsync 0.1.5

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
+build_dir="${__proj_dir}/build"
+go_build=(go build -ldflags "-w")
 
-# Make dir
-mkdir -p $ROOTFS
+_info "project path: ${__proj_dir}"
+_info "plugin name: ${plugin_name}"
+
+# Disable CGO for builds
+export CGO_ENABLED=0
```
Comment (kindermoumoute) [inline scripts/build.sh]: This flag was previously set to 1 on this file. Any idea why?

Comment (nanliu) [inline scripts/build.sh]: Cody saw this too, I think it's a mistake because it's not in sbr and it builds correctly.

**Comment by kindermoumoute** [comment]
LGTM.

File: scripts/build.sh (hunk 1 above)
Comment (sandlbn) [inline scripts/build.sh]: Yes, that was mistake.

**Pull request merged**